src/wes.c

```diff
@@ -29,9 +29,33 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 #if defined(_WIN32)
     #include <windows.h>
+
+#ifdef WINAPI_FAMILY
+#if (!WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) && WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP))
+#define XASH_WINRT
+#endif
+#endif
+
+#ifdef XASH_WINRT
+static void* dlopen(const char* dllname)

```

Comment:
dlopen has second argument by standard.
`void *dlopen(const char *filename, int flags);`

Reply:
Also, even if wesgl is worst formatted project we have, keep indentation please. :)